### DISCUSSION TITLE
Shared expenses

### DISCUSSION THREAD

**Discussion by gbcatrinoiu** [ellite/Wallos#614] [Ideas]
I would like to propose a new feature for shared expenses that would help users keep track of common costs. The feature would allow each user to add payments as usual, with an additional field in the form where other users can be included.

The subscriptions dashboard would also display shared expenses in a separate section, allowing users to easily see which subscription is paid by whom.

Additionally, the statistics dashboard would have a new section for shared expenses, showing how much users owe to each other.

**Comment by Geekujin** [comment]
I love this idea. I think a 'Household' subscriptions, visible to everyone deemed as responsible for household bills via an admin setting (e.g. all family members or just parents) would be great for managing shared expenses. It would also be nice to multi-select when choosing the 'Paid by' user, or have the ability to add a 'money owed' type expense to another household members subscriptions if they contribute to it (for example kids mobile phone bills or something).

ETA: I currently get round the shared expenses by having a 3rd account with shared password just for joint subs, although it's a bit unwieldy.
